README.md

```diff
@@ -7,3 +7,4 @@ Silakan gunakan format berikut:<br>
 
 Berikut adalah daftar Volunter yang diterima:
 * Oon Arfiandwi, [oo.or.id](https://oo.or.id).
+* Fido Jahfal, [Linkedin](https://www.linkedin.com/in/fidojahfal)
```

Comment:
Hello @jimanisothername 
I like your portofolio,
welcome to our project
Thank you so much!